.github/workflows/ci.yml

```
@@ -9,11 +9,11 @@ on:
 jobs:
   build:
 
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
 
     strategy:
       matrix:
-        node-version: [14.x, 16.x, 17.x]
+        node-version: [14.x, 16.x, 18.x]
```

```suggestion
        node-version: [14.x, 16.x, 17.x, 18.x]
```
Nodejs 17 is the current version. Is this being skipped?

---

`18.3.0` is current. https://nodejs.org/en/

---

Right you are, outdated link on my end.